jenkins-scripts/dsl/ignition.dsl

```diff
@@ -80,6 +80,7 @@ ignition_colcon_win         = [ 'gazebo',
                                 'physics',
                                 'rendering',
                                 'sensors',
+                                'transport',
```

Comment:
Adding transport here is the right thing, but name is going to change to get the shorter version. Just FYI.

Reply:
ok, we'll need to update the ignition_collection dsl; I'll do that in a follow-up